e2e/api/steps.js

```diff
@@ -823,6 +826,33 @@ async function updateCaseDataWithPlaceholders(data, document) {
   return JSON.parse(data);
 }
 
+// CIV-5514: remove when hnl is live
+async function replaceWitnessIfHNLFlagIsDisabled(defendantResponseData, solicitor) {
+  let isHNLEnabled = await checkToggleEnabled('hearing-and-listing-sdo');
+  // work around for the api  tests
+  console.log(`Witness selected in Env: ${config.runningEnv}`);
+  if (!isHNLEnabled || !(['preview', 'demo'].includes(config.runningEnv))) {
```

Comment:
Same as the comment left in witnesses page. Should this be getting replaced in preview if we exclude the changes.